pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -40,8 +40,10 @@
 )
 
 from pyomo.common.dependencies import attempt_import
+import numpy as np
+import struct
 
-gdxcc, gdxcc_available = attempt_import('gdxcc')
+gdxcc, gdxcc_available = attempt_import('gams.core.gdx') 
```

Comment:
I would suggest the following change to support both pre-GAMS-45 and current GAMS APIs:
```python
def _gams_importer():
    try:
        import gams.core.gdx as gdx
        return gdx
    except ImportError:
        try:
            # fall back to the pre-GAMS-45.0 API
            import gdxcc
            return gdxcc
        except:
            # suppress the error from the old API and reraise the current API import error
            pass
        raise

gdxcc, gdxcc_available = attempt_import('gdxcc', importer=_gams_importer) 
```